crates/rspack_core/src/runtime_globals.rs

```diff
@@ -245,6 +245,9 @@ bitflags! {
     const PRELOAD_CHUNK = 1 << 60;
 
     const PRELOAD_CHUNK_HANDLERS = 1 << 61;
+
+    // rspack only
+    const RSPACK_VERSION = 1 << 62;
   }
```

Comment:
will it conflict with future webpack new runtime_globals?

Reply:
I think not, if webpack use `__webpack_require__.rv`, we should just change to another. It only affects the result of code generation, users will not feel the change.